04-Protocol-Security.md

```diff
@@ -77,8 +77,10 @@ CipherState has the following interface:
   * Sets `k = key`, `n = 0`
 * `EncryptWithAd(ad, plaintext)`
   * if `k` is non-empty, performs `ENCRYPT(k, n++, ad, plaintext)` on the underlying cipher function, otherwise returns `plaintext`
```

Comment:
Looks great. Super nit, but do we want to capitalize the "i" in the first "if" of this starting line for consistency?

Reply:
good point.